Robust.Shared/Serialization/Manager/Result/DeserializationResult.cs

```diff
@@ -33,6 +19,6 @@ public static DeserializationResult Value(Type type, object? value)
 
     public abstract class DeserializationResult<T> : DeserializationResult
     {
-        public abstract T Value { get; }
+        public abstract T? Value { get; }
```

Comment:
you putting nullables (explanation below image, meme was too big, pls scroll)
![image](https://user-images.githubusercontent.com/16618384/132878902-b9d5de83-ef11-4dd3-a66b-5c40772bee05.png)

this should in every case of the good lord be `T`, not `T?`. If i do a `DeserializationResult<int>`, i do not want value to be `int?`, i want it to be `int`. Now if i do `DeserializationResult<int?>` it will be `int?`.
```suggestion
        public abstract T Value { get; }
```

Reply:
Fixed in https://github.com/space-wizards/RobustToolbox/pull/2018/commits/0d2ec32d0fb6ee2ba845024940728ff26f0804ad